docs/make.jl

```diff
@@ -46,9 +46,10 @@ mathengine = MathJax(Dict(
 ))
 
 format = Documenter.HTML(
-    prettyurls = get(ENV, "CI", "") == "true",
+    prettyurls = get(ENV, "CI", "") != "" ? true : false,
```

Comment:
```suggestion
    prettyurls = !isempty(get(ENV, "CI", "")),
```
Maybe a bit simpler to parse?